backend/tests/icpy/agent/tools/test_atlascloud_ttv_tool.py

```
@@ -0,0 +1,467 @@
+        """Test saving video to local workspace."""
+        prompt = "Test video prompt"
+        model = "bytedance/seedance-v1-lite-t2v-480p"
+        
+        mock_context = {'contextId': 'local'}
+        mock_filesystem = MagicMock()
+        mock_filesystem.root_path = "/workspace"
+        
+        with patch('icpy.agent.tools.context_helpers.get_current_context', return_value=mock_context), \
+             patch('icpy.agent.tools.context_helpers.get_contextual_filesystem', return_value=mock_filesystem), \
+             patch('os.makedirs'), \
+             patch('builtins.open', mock_open()) as mock_file:
```

Ruff (0.14.14)

[error] Local variable `mock_file` is assigned to but never used

Remove assignment to unused variable `mock_file`

(F841)